docs/docs/tutorials/tutorial_example/FashionMNIST/train.py

```diff
@@ -0,0 +1,70 @@
+import os
+import argparse
+
+import torch
+import torchvision
+import torch.nn as nn
+import torchvision.transforms as transforms
+
+from bigdl.orca import init_orca_context, stop_orca_context
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.metrics import Accuracy
+from bigdl.orca.data.file import get_remote_file_to_local
+
+parser = argparse.ArgumentParser(description='PyTorch Example')
+parser.add_argument('--cluster_mode', type=str, default="spark-submit",
+                    help='The cluster mode, such as local, yarn-client, yarn-cluster, spark-submit or k8s.')
+parser.add_argument('--remote_dir', type=str, help='The path to load data from remote resources like HDFS or S3')
+args = parser.parse_args()
+
+
+def train_data_creator(config, batch_size):
+    transform = transforms.Compose([transforms.ToTensor(),
+                                transforms.Normalize((0.5,), (0.5,))])
+    if args.remote_dir is not None:
+        data_dir = "/tmp/dataset"
+        get_remote_file_to_local(remote_path=args.remote_dir, local_path=data_dir)
+        trainset = torchvision.datasets.FashionMNIST(root=data_dir, train=True,
+                                                     download=False, transform=transform)
+        trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size,
+                                                  shuffle=True, num_workers=0)
+    else:
+        print("Please specify the dataset path on a network file system.")
+    return trainloader
+
+def main():
+    if args.cluster_mode.startswith("yarn"):
+        if args.cluster_mode == "yarn-client":
+            init_orca_context(cluster_mode="yarn-client", cores=4, memory="10g", num_nodes=2,
```

Comment:
you can merge yarn-client and cluster into one init_orca_context,  `cluster_mode=args.custer_mode`